scripts/fwup.nuspec

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>fwup</id>
+    <title>fwup</title>
+    <version>0.9.1</version>
```

Comment:
Do you agree that it would be better to dynamically populate this version from [the VERSION file](https://github.com/fhunleth/fwup/blob/master/VERSION) at build time so we wouldn't have to remember to update it when packaging a new build?

For example, you could put `%VERSION%` in the `<version>` tag in the checked-in `fwup.nuspec` file, and have a step in the `build_pkg.sh` script, just before the `choco.exe pack` command, that would run `sed -i "s/%VERSION%/$(cat VERSION)/" fwup.nuspec` to finalize the metadata before using it.